install/install.sh

```diff
@@ -229,4 +234,4 @@ echo "You can access after the reboot:"
 echo "- The computer webpage: http://blueos.local"
 echo "- The ssh client: pi@blueos.local"
 echo "System will reboot in 10 seconds."
-sleep 10 && reboot
```

Comment:
No need to commit this change.

PS: our linter should be getting those cases, but it isn't.

Reply:
I didn't mean to make that change... I don't know what you mean about the linter.

Reply:
This was probably an auto-formatting from your IDE. Are you on Windows? If I remember correctly it does this because the line ending character is different. You can just unstage this before commiting.

About the linter, I'm just noticing that the linter we have in the repository should be getting those cases automatically.

Reply:
Yep. I'm in VSCode on Windows.

No, it's not the different line ending - I just have it set up to add a newline at the end of the file. Most of the scripts in the repo *do* have a newline at the end, and I'm surprised this one doesn't and that your linter doesn't enforce it!